pkg/portableresources/backend/controller/createorupdateresource.go

```diff
@@ -177,7 +177,7 @@ func (c *CreateOrUpdateResource[P, T]) executeRecipeIfNeeded(ctx context.Context
 	if err != nil {
 		return nil, fmt.Errorf("failed to get connected resource IDs: %w", err)
 	}
-	connectedResourcesProperties := make(map[string]map[string]any)
+	connectedResourcesProperties := make(map[string]recipes.ConnectedResource)
```

Comment:
Can this be renamed to just `connectedResources`? Do we need to have `Properties` postfix?